docs/admin/multiple-zones.md

```diff
@@ -0,0 +1,280 @@
+---
+---
+
+## Introduction
+
+Kubernetes 1.2 adds support for running a single cluster in multiple zones.
+This is a lightweight version of a broader effort for federating multiple
+Kubernetes clusters together (sometimes referred to by the affectionate
+nickname "Ubernetes").  Full federation will allow combining separate
+Kubernetes clusters running in different regions or clouds.  However, many
+users simply want to run a more available Kubernetes cluster in multiple zones
+of their cloud provider, and this is what the multizone support in 1.2 allows
+(we nickname this "Ubernetes Lite").
+
+Multizone support is deliberately limited: a single Kubernetes cluster can run
+in multiple zones, but only within the same region (and cloud provider).  Only
+GCE and AWS are currently supported automatically (though it would be easy to
+add similar support for other clouds or even bare metal, by simply arranging
+for the appropriate labels to be added to nodes and volumes).
+
+
+* TOC
+{:toc}
+
+## Functionality
+
+When nodes are started, the kubelet will automatically add labels to them with
+zone information.
+
+Just as pods in a replication controller or service are spread across nodes,
```

Comment:
The wording of this sentence is confusing.  Suggest rewording to something like:

"Just as the pods in a replication controller or service are automatically spread across nodes in a single-zone cluster (to reduce the impact of node failures), this spreading behavior is extended across zones in a multi-zone cluster (to reduce the impact of zone failures).  This is achieved via `SelectorSpreadPriority`.